MiniApp/Classes/Protocols/MiniAppAuthenticationDelegate.swift

```diff
@@ -0,0 +1,24 @@
+/**
+ Public Protocol that will be used by the Mini App to get Authentication
+ info, such as Access token
+*/
+public protocol MiniAppAuthenticationDelegate: class {
+    /// Interface that is used to retrieve the Token Info
+    func getAccessToken(completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void)
```

Comment:
We need to also pass the mini app ID to this method. This is so the Host App can verify that this mini app should be able to use the access token. See the android implementation: 
https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt#L30

Reply:
Done 👍